know/rdf/parser/summarizer.py

```diff
@@ -29,3 +32,18 @@ def valid_viaf(self):
             Otherwise returns false
         """
         return 'viaf' in self.uri
+
+    def get_wikidata_uri(self):
+        """ Parse viaf rdf file and search for a wikidata
+            resource that is the 'sameAs' the given one.
+        """
+        graph = rdflib.Graph()
+        graph.parse(self.uri)
```

Comment:
Can you add a comment explaining what `graph.parse` does? Kind of weird seeing it being called yet you don't do anything with a returned variable or anything